dropwizard-service-discovery-bundle/src/main/java/io/appform/dropwizard/discovery/bundle/id/IdGenerator.java

```diff
@@ -65,7 +65,7 @@ public IdInfo(int exponent, long time) {
     private static List<IdValidationConstraint> globalConstraints = Collections.emptyList();
     private static Map<String, List<IdValidationConstraint>> domainSpecificConstraints = new HashMap<>();
     private static final Retryer<GenerationResult> retrier = RetryerBuilder.<GenerationResult>newBuilder()
-            .withStopStrategy(StopStrategies.stopAfterAttempt(512))
+            .withStopStrategy(StopStrategies.stopAfterAttempt(Integer.parseInt(System.getenv().getOrDefault("NUM_ID_GENERATION_RETRIES", "512"))))
```

Comment:
Let's add test cases for this

Reply:
This cannot be tested as seting env variable is difficult in junit